test/compose/simple_port_map/frontend/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine
 WORKDIR /app
-RUN apk update && apk add py3-pip && pip3 install flask
+RUN apk update && apk add py3-flask
```

Comment:
`pip3 install flask` is giving a `This environment is externally managed` error since Python 3.11

https://github.com/pypa/pip/issues/12429
https://peps.python.org/pep-0668/